downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -54,6 +54,17 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 - Introduced Avro serialization/deserialization support
 
+#### `graphql` package
+
+- Introduced GraphQL server-side caching support
+
+### Improvements
+
+#### `graphql` package
+
+- Use aliases instead of field names in GraphQL error responses
+- Reporting GraphQL specific diagnostics in the VS Code extension
```

Comment:
Can we use the past tense here to align it with the other notes?

Reply:
Fixed with 61b112c31e0c799c56cbae015ff7c5dd629afbb1